tests/robot-tests/tests/seed_data/seed_data_generation_mechanism.robot

```diff
@@ -22,13 +22,13 @@ Force Tags          Local    Dev
 
 
 *** Variables ***
-${PUBLICATION_NAME}     Seed data mechanism publication %{RUN_IDENTIFIER}
+${PUBLICATION_NAME}     UI tests - seed data mechanism publication %{RUN_IDENTIFIER}
```

Comment:
So I think adding "UI tests" here means that this will be deleted when we trigger the endpoint to clean up test themes?

Pretty sure this is deliberate, but just double checking.

Reply:
Yes that's right!